src/router/index.js

```diff
@@ -0,0 +1,40 @@
+/*
+//  index.js
+//
+//  Created by Kalila L. on May 9th, 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+*/
+
+import { route } from 'quasar/wrappers'
```

Comment:
Single quotes and missing ";"s in this file and other JavaScript files.

Reply:
Actually, I should note that I've for a long time converted to using only single quotes. The ';'s are still up in the air though. The reason for doing this is because the ESLint config on the dashboard was defaulted that way and I picked it up as a good habit anyway.

The reason being is that double-quotes are sometimes necessary if you're assembling code within code or working with HTML. So, using single-quotes by default will reduce the need to \ out your quotes.

Reply:
OK, single quotes make sense for HTML-related JavaScript, and working in with components from Quasar/Vue libraries.
Don't know about semicolons though - may depend on how much components / whatever come in from Quasar/Vue libraries.